mlflow/models/evaluation/lift_curve.py

```diff
@@ -0,0 +1,153 @@
+import matplotlib.pyplot as plt
+
+import numpy as np
+
+
+def cumulative_gain_curve(y_true, y_score, pos_label=None):
```

Comment:
Since we're only using this for `lift_curve`, can we make it "private"?

```suggestion
def _cumulative_gain_curve(y_true, y_score, pos_label=None):
```